tools/crm_error.c

```diff
@@ -140,8 +65,8 @@ main(int argc, char **argv)
 {
     crm_exit_t exit_code = CRM_EX_OK;
     int rc = pcmk_rc_ok;
-    const char *name = NULL;
-    const char *desc = NULL;
+
+    pcmk_result_type_t type = pcmk_result_legacy;
```

Comment:
This is the current default. We need to figure out when and how we want to change it without introducing an unplanned regression for anything external that might depend on `crm_error`.